org.gtk.Gtk3theme.Obsidian-2-Aqua.json

```diff
@@ -0,0 +1,46 @@
+{
+   "id":"org.gtk.Gtk3theme.Obsidian-2-Aqua",
+   "branch":"3.22",
+   "runtime":"org.freedesktop.Sdk",
+   "build-extension":true,
+   "sdk":"org.freedesktop.Sdk",
+   "runtime-version":"19.08",
```

Comment:
Can you change it and other variants to 20.08?

Reply:
Sure. Done.